vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -34,23 +34,14 @@ switch ($windowsSKU) {
             "mcr.microsoft.com/windows/servercore:ltsc2019",
             "mcr.microsoft.com/windows/nanoserver:1809",
             "mcr.microsoft.com/oss/kubernetes/pause:3.4.1",
-            "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.2.0",
             "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.3.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.1.0",
             "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.2.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.2.0",
             "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.4.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.2.0",
             "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.4.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v0.0.19",
             "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v0.0.21",
-            "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:0.0.12",
             "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:0.0.14",
-            "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v0.5.1", # for k8s 1.18.x
             "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v0.6.0", # for k8s 1.19.x
-            "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v0.7.4", # for k8s 1.20.x
             "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v0.7.5", # for k8s 1.20.x
```

Comment:
can we make it v0.7.6?

Reply:
Sure. This PR is used to cleanup old images and packages. Please free to create a PR to add new versions.